.travis.yml

```diff
@@ -42,6 +42,7 @@ matrix:
         # Sphinx is pinned so that new versions that introduce new warnings won't suddenly cause build failures.
         # (Updating the version is fine as long as no warnings are raised by doing so.)
         - python -m pip install sphinx~=1.6.1
+        - python -m pip install blurb
```

Comment:
@larryhastings did you push this version? If so, was keeping the dev version on purpose or an accident? If the latter we can just fix the version number and do another release as 1.0.2 so the dependency can be `>=1.0.2`.